### ISSUE TITLE
Prediction on Image and mask without UI

### ISSUE THREAD

**Issue by kashyappiyush1998** [zsyzzsoft/co-mod-gan#4]
I have a binary mask and image to be filled and want to predict output, how do I do that using this repository. Like in deep fill v1/v2 we can predict it directly. Thank you.
Results are pretty awesome on comodgan.ml

**Comment by styler00dollar** [comment]
I agree. I would be interested to use co-mod-gan inside Google Colab and that needs command line. I tried to modify ``run_demo.py``, but I got many errors.

**Comment by zsyzzsoft** [comment]
Now one can create a custom dataset with `dataset_tools/create_from_images` and then run demo with it!

**Comment by kashyappiyush1998** [comment]
@zsyzzsoft  thanks for this added tool, but is it not possible to directly get the predicted image out of the image and mask without running UI.

**Comment by zsyzzsoft** [comment]
> @zsyzzsoft thanks for this added tool, but is it not possible to directly get the predicted image out of the image and mask without running UI. If possible, I have one more request, can you provide a sample of inputs both image and mask along with it.
> 
> Thanks.

Please try the newly added `run_generator.py`.

**Comment by styler00dollar** [comment]
Thanks, it works.

**Comment by tiwarikaran** [comment]
@styler00dollar hi can you share the notebook if possible I am having a tough time implementing this

**Comment by styler00dollar** [comment]
@tiwarikaran Well I am pretty late, but my colab notebook is [here](https://github.com/styler00dollar/Colab-co-mod-gan/blob/master/Colab-co-mod-gan.ipynb).
